scripts/build-color-variants.js

```diff
@@ -208,15 +220,18 @@ function buildColorVariants(variables, config) {
       return (result += stripIndent(`
         .textarea--border-${color},
         .input--border-${color},
-        .select--border-${color} {
+        .select--stroke.select--${color} {
           box-shadow: inset 0 0 0 1px var(--${color});
         }
 
         .textarea--border-${color}:focus,
-        .input--border-${color}:focus,
-        .select--border-${color}:focus {
+        .input--border-${color}:focus {
           box-shadow: inset 0 0 0 1px var(--${darkerShade});
         }
+
+        .select--stroke.select--${color}:focus {
+          box-shadow: inset 0 0 0 1px var(--${darkerShade}), var(--focus-shadow);
+        }
```

Comment:
We only want to use the double-shadow when `[data-assembly-focus-control='visible']` selector is true, so this should be:

```css
[data-assembly-focus-control='visible'] .select--stroke.select--${color}:focus {
  box-shadow: inset 0 0 0 1px var(--${darkerShade}), var(--focus-shadow);
}
```